src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -10,10 +10,10 @@ public class RacingCarController {
     private final RacingGame racingGame = new RacingGame(new RacingCarTable());
 
     public void run() {
-        int numberOfCars = RetryView.retryIfNotValidNumber(InputView::inputNumberOfCars);
+        String carNames = RetryView.retryIfNotValidNames(InputView::inputCarNames);
         int tryCount = RetryView.retryIfNotValidNumber(InputView::inputTryCount);
```

Comment:
DTO를 활용한다면 이렇게 할 수도 있을 것 같아요.
```suggestion
        RacingRequest request = racingInputView.input();
```
최대한 핵심업무를 제외한 구체화에 대한 의존성을 줄이는거죠.